tools/privacyidea-token-janitor

```diff
@@ -296,6 +306,12 @@ def _get_tokenlist(last_auth, assigned, active, tokeninfo_key,
             if description and not re.search(description,
                                              token_obj.token.description):
                 continue
+            if has_not_tokeninfo_key:
+                if token_obj.get_tokeninfo(has_not_tokeninfo_key) is not None:
```

Comment:
If the value of the tokeninfokey is None, than the logic does no work, imho.

It might be better to check:

      if has_not_tokeninfo_key not in token_obj.get_tokeninfo()

to check, if the key is not contained in the dict.

Reply:
@NNLemling please fix this.

Reply:
I tested this change and it doesn't work. I will see what it is tomorrow. All further changes for this issue will be made here: #2884